lib/elixir_google_scraper/scraper/search_result.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ElixirGoogleScraper.Scraper.SearchResult do
```

Comment:
Maybe moved to `/schemas/SearchResult.ex` like the `keyword` schema and add tests for changeset 👍